aioredis/pool.py

```diff
@@ -394,6 +394,9 @@ def _fill_free(self, *, override_min):
             self._acquiring += 1
             try:
                 conn = yield from self._create_new_connection(address)
+                # check the healthy of that connection, if
+                # something went wrong just trigger the Exception
+                yield from conn.execute('ping')
```

Comment:
No really happy with that change - to much implicit -, but it is needed a deterministic way to check that the pool can be created with that min_size requirements.  Should we keep this way to check that a connection is not really opened or try something more sophisticated (hard problem)?

Also have in mind that the code that is above when the kw `override_min` is True, should be following the same pattern than here?